src/main/java/com/github/manolo8/darkbot/gui/utils/table/FormationNpcInfoEditor.java

```diff
@@ -0,0 +1,99 @@
+package com.github.manolo8.darkbot.gui.utils.table;
+
+import com.github.manolo8.darkbot.gui.utils.PopupMenuListenerAdapter;
+import eu.darkbot.api.game.items.SelectableItem;
+
+import javax.swing.JLabel;
+import javax.swing.JMenuItem;
+import javax.swing.JPopupMenu;
+import javax.swing.JTable;
+import javax.swing.SwingUtilities;
+import javax.swing.ToolTipManager;
+import javax.swing.event.PopupMenuEvent;
+
+import java.awt.Rectangle;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class FormationNpcInfoEditor extends TableDelegateEditor<JLabel> {
```

Comment:
similar as pointed-out above, maybe we need icons and not text to show these, that would make them as compact as prior editor, where it was just a char plus the "key" icon